### PR TITLE
Dropdown vs radiobuttons

### PR DESCRIPTION
**Pull request by 1903Daniel**
Fixes #

Please make sure you conform to the contributor guidelines and agreed to the contributor license agreement, see https://github.com/eclipsesource/jsonforms/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/core/testers.ts [hunk 1]
```diff
@@ -18,6 +18,40 @@ export const isControl = (uischema: any): uischema is ControlElement =>
...
 
 
+/**
+ * Checks whether the given UI schema has an option with an enum array
+ * and whether the length of the enum is at least the given minLength.
+ * If no enum array is present, returns false.
+ *
+ * @param {number} minLength the lower bound for the length of the enum array
+ */
+export const enumLengthAtLeast = (minLength: number): Tester =>
```
Comment (eneufeld) [inline src/core/testers.ts]: This needs tests in the core package.

File: src/renderers/materialized/materialized.radiobutton.control.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                        return (
+                            <p>
+                                <input
+                                    type='radio'
+                                    name={label}
+                                    id={optionValue}
+                                    value={optionValue}
+                                />
+                                <label for={optionValue}
+                                       onClick={e => this.handleChange(e.target.value)}>
```
Comment (edgarmueller) [inline src/renderers/materialized/materialized.radiobutton.control.tsx]: Data binding doesn't work here, I think `onChange` on input should work though (like in the default radiobutton renderer)

File: src/core/testers.ts [hunk 3]
```diff
@@ -18,6 +18,40 @@ export const isControl = (uischema: any): uischema is ControlElement =>
...
+    };
+
+/**
+ * Checks whether the given UI schema has an option with an enum array
+ * and whether the length of the enum is at most the given maxLength.
+ * If no enum array is present, returns false.
+ *
+ * @param {number} maxLength the upper bound for the length of the enum array
+ */
+export const enumLengthAtMost = (maxLength: number): Tester =>
```
Comment (eneufeld) [inline src/core/testers.ts]: @edgarmueller do you prefer two different methods, or should we rather extend the `isEnumControl` tester and add two parameters, where the first is the `minLength` and the second is `maxLength`?

File: test/renderers/radiobutton.control.test.tsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                    }
+                }
+            }
+        ),
+        2
+    );
+});
+
+test('tester with matching numeric type', t => {
+    // TODO should this be true?
```
Comment (edgarmueller) [inline test/renderers/radiobutton.control.test.tsx]: Comment can be removed

File: src/core/testers.ts (hunk 3 above)
Comment (edgarmueller) [reply src/core/testers.ts]: I like the latter more

**Comment by 1903Daniel** [comment]
currently under re-implementation for material UI

**Pull request closed without merging**